index.js

```diff
@@ -116,6 +117,7 @@ function do_bundle(file, opts){
                 if( opts.arithmetic ) {
                     jspm_input += ' ' + opts.arithmetic.trim();
                 }
+                jspm_input = jspm_input.replace(/\\/g, '/');
```

Comment:
It looks like JSPM expects paths to be in URL form. My workaround is to only do the substitution on windows by detecting the path separator. see https://github.com/systemjs/builder/issues/241 for seemingly related discussion